mande/assetlist.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "mande",
+  "assets": [
+    {
+      "description": "The native token of Mande Network.",
+      "extended_description": "Credibility Hub for Web3",
+      "denom_units": [
+        {
+          "denom": "amand",
+          "exponent": 0
+        },
+        {
+          "denom": "mand",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "amand",
+      "name": "Mande Network",
+      "display": "mand",
+      "symbol": "MAND",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mande/images/mande.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mande/images/mande.svg"
+        }
+      ],
+      "socials": {
+        "website": "https://mande.network/",
+        "twitter": "https://twitter.com/MandeNetwork"
+      },
+      "keywords": [
+        "credibility",
+        "web3"
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mande/images/mande.png",
```

Comment:
Can I request the png so be made a bit smaller to reduce it's file size? ~3000x3000 is quite huge....
Also, can we please put this in a square canvas so the image is 1:1 AR?

Reply:
Done, added 500X500